docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,118 @@
+---
+---
+
+* TOC
+{:toc}
+
+
+## Prerequisites
+
+The only thing you need is a linux machine with **Docker 1.10.0 or higher**
+
+## Overview
+
+This guide will set up a 2-node Kubernetes cluster, consisting of a _master_ node which hosts the API server and orchestrates work
+and a _worker_ node which receives work from the master. You can repeat the process of adding worker nodes an arbitrary number of
```

Comment:
For better or worse we have settled on using the terms "master" and "node". It's confusing to keep using worker and minion in documentation.